### PR TITLE
DP-2330 Enhanced descriptions for Python SDK methods

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/docs/hugo/docs/content/en/docs/administration/users/get_user.md [hunk 1]
```diff
@@ -6,16 +6,25 @@ no_list: true
 superheading: "catalog_user."
 ---
 
-<!-- TODO -->
+
 
 ``get_user(user_id: str)``
 
-Returns *CatalogUser*.
+Gets an individual user using User id.
```
Comment (hkad98) [inline gooddata-sdk/docs/hugo/docs/content/en/docs/administration/users/get_user.md:13]: Why is there a User and not a user? Below, there are examples e.g. data sources that are all chars lowercase. Could we unify this, please?

File: gooddata-sdk/gooddata_sdk/insight.py [hunk 2]
```diff
@@ -495,10 +499,15 @@ def get_insight(self, workspace_id: str, insight_id: str) -> Insight:
         """
         Gets a single insight from a workspace.
 
-        :param workspace_id: identifier of workspace to load insight from
-        :param insight_id: identifier of the insight
-        :return: single insight; the insight will contain sideloaded metadata about the entities it references
-        :rtype: Insight
+        Args:
```
Comment (hkad98) [inline gooddata-sdk/gooddata_sdk/insight.py:502]: Unify spaces, please. See the example above.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

File: gooddata-sdk/docs/hugo/docs/content/en/docs/administration/user-groups/delete_user_group.md [hunk 3]
```diff
@@ -6,11 +6,20 @@ no_list: true
 superheading: "catalog_user."
 ---
 
-<!-- TODO -->
+
 
 ``delete_user_group(user_group_id: str)``
 
-Delete a user group.
+Deletes User Group using User Group id.
```
Comment (hkad98) [inline gooddata-sdk/docs/hugo/docs/content/en/docs/administration/user-groups/delete_user_group.md:13]: Please, fix also here the uppercase for user group.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR202

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/33/ : SUCCESS in 2m 02s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/183/ : SUCCESS in 2m 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/186/ : SUCCESS in 21s

**Pull request merged**